test/functional/test.sh

```diff
@@ -59,6 +59,35 @@ do
     trap - EXIT
 done
 
+for siteInfo in $(cat test_template_sites);
+do
+    trap cleanup_initialized EXIT
+
+    flag=$(cut -d',' -f1 <<<"$siteInfo")
+    site=$(cut -d',' -f2 <<<"$siteInfo")
```

Comment:
`test_sites` and `test_convert_sites` don't use `,` as delimiters, instead each entry is a single line. Let's also do that for `test_template_sites`.

Reply:
I am not very sure about this. I see it as 1 test case is 1 single line. 

Suppose I remove `,`  I either derive the flag from the path. For example:

1. `test_site_templates/test_minimal` -> minimal
1. Or derive the path `minimal` -> `test_site_templates/test_minimal`.

Between these two, I would prefer the second one, but when writing this, I was thinking that being explicit between the flag and the test path would have been better. What do you think?

Reply:
Ah icic, had some misunderstanding about this part of the code, I am fine with this approach. :+1: